invenio_records_resources/resources/record.py

```diff
@@ -51,15 +74,16 @@ def __init__(
     #
     def search(self, *args, **kwargs):
         """Perform a search over the items."""
+        # TODO fix identity extraction
         identity = g.identity
-        record_search = self.service_cls.search(
-            querystring=resource_requestctx.request_args.get("q", ""),
-            identity=identity,
-            pagination=resource_requestctx.request_args.get("pagination"),
+        return (
```

Comment:
Minor: I'm fine with this change. But it feels changing code that is totally equivalent to the previous for a mere "programming" preference. Might affect history.

Reply:
I did it just to normalize all the views as we had 2 ways of returning the view's result :) It's one less line though...But I am fine to change it back if you see it as problematic!

Reply:
I am with Pablo on this one :)